Round/shared.css

```diff
@@ -288,6 +294,14 @@ div.basiccontextmenu_contextMenuContents_TBSbv  {
   + .basiccontextmenu_contextMenuItem_3PqLg,
 .Install
   ~ .basiccontextmenu_ContextMenuSeparator_zVMb1
-  + .basiccontextmenu_contextMenuItem_3PqLg {
+  + .basiccontextmenu_contextMenuItem_3PqLg,
+.basiccontextmenu_ContextMenuSeparator_zVMb1
+  + .basiccontextmenu_contextMenuItem_3PqLg:nth-last-child(3):nth-child(7),
+.basiccontextmenu_ContextMenuSeparator_zVMb1
+  + .basiccontextmenu_contextMenuItem_3PqLg:last-child:nth-child(9),
+.basiccontextmenu_ContextMenuSeparator_zVMb1
+  + .basiccontextmenu_contextMenuItem_3PqLg:nth-last-child(3):nth-child(6),
+.basiccontextmenu_ContextMenuSeparator_zVMb1
+  + .basiccontextmenu_contextMenuItem_3PqLg:last-child:nth-child(8) {
   border-radius: 20px;
 }
```

Comment:
You have to change it when:

- Play Button is Shown
- Install Button is Shown
- Steam Game
- Non-Steam Game

All of which have different ways to access the dom item with out messing with anything else in the ui.